tips/githubTips.md

```diff
@@ -1,21 +1,21 @@
-## GitHub Tips
+**GitHub Tips**
```

Comment:
This is a main heading, but will look the same as the other headings. I think if the markdown is rendered correctly it will look better than the un-rendered markdown - so lets give it a try 👍